terraform/kubeflow-gateway.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
Successfully deployed to classic and VPC clusters. So everything looks good. One question here for my understanding, where is this file used? And why there is a fixed host name?

Reply:
it's not used now. originally, I want to use this in the ansible playbook. I embedded it into the ansible playbook. I can remove it.